sdk/python/feast/permissions/auth/oidc_token_parser.py

```diff
@@ -68,12 +68,17 @@ async def user_details_from_access_token(self, access_token: str) -> User:
 
         try:
             signing_key = jwks_client.get_signing_key_from_jwt(access_token)
+            ## TODO: Do we need to inject these from config or fixed?
```

Comment:
IMO, for now it can be fixed. We're not going to fully support other auth managers apart from K8s (better, OCP)